runtime/altair/src/lib.rs

```diff
@@ -1977,6 +2001,7 @@ construct_runtime!(
 		Treasury: pallet_treasury::{Pallet, Call, Storage, Config<T>, Event<T>} = 69,
 		Preimage: pallet_preimage::{Pallet, Call, Storage, Event<T>} = 70,
 		Uniques: pallet_uniques::{Pallet, Call, Storage, Event<T>} = 72,
+		Paras: polkadot_runtime_parachains::paras::{Pallet, Call, Storage, Event, Config<T>, ValidateUnsigned} = 73,
```

Comment:
This test here - https://github.com/centrifuge/centrifuge-chain/blob/polkadot-v1.1.0-it-env-fix/runtime/integration-tests/src/generic/cases/liquidity_pools.rs#L7316

```
  _config: Default::default(),
     |                              ^^^^^^^^^^^^^^^^^^ the trait `polkadot_runtime_parachains::paras::Config` is not implemented for `T`
 ```
 
 I'll try removing that from the test and see if we can skip adding this, I don't remember the setups required for most of these tests now.

Reply:
We're no longer able to open an HRMP channel for XCM messages if we remove that `GenesisConfig`:

```
Expected Ok(_). Got Err(
    DispatchErrorWithPostInfo {
        post_info: PostDispatchInfo {
            actual_weight: None,
            pays_fee: Pays::Yes,
        },
        error: Module(
            ModuleError {
                index: 60,
                error: [
                    1,
                    0,
                    0,
                    0,
                ],
                message: Some(
                    "OpenHrmpChannelInvalidRecipient",
                ),
            },
        ),
    },
)
```

Reply:
Is this not also on Centrifug? Then why do we want to clean that up?

Reply:
@mustermeiszer I specifically added that to all runtimes in this PR.

Reply:
There must be another way without adding this pallet. It is meant to be used on relay chains.

Reply:
OK, then I'll do a bit of digging there. Thanks for checking!

Reply:
On [this line](https://github.com/centrifuge/centrifuge-chain/blob/polkadot-v1.1.0-it-env-fix/runtime/integration-tests/src/generic/cases/liquidity_pools.rs#L7316), I should've used `FudgeRelayRuntime<T>` instead of `T`. 

Now it works and I will go ahead and remove the extra pallet. 

Thank you all for checking this!!! :heart:

Reply:
Thanks for look deeply into this!